packages/gatsby-theme-newrelic/src/utils/track.js

```diff
@@ -0,0 +1,13 @@
+function track(fn, handler, before) {
```

Comment:
I'm a little unsure what this method is meant for. `track` makes me think its going to track something in like google analytics, or tessen or something, but it looks like maybe this is just a result of calling multiple functions. Could you talk through the idea behind this?

Reply:
With the other suggestions mentioned above, you might not need this function anymore. If those other suggestions work for you, you can remove this.

Reply:
I'm going to look into using navigate and see if it solves my problem, but the current issue has to do more with the interactions between the `search-ui` changing the URL and then having us open the URL when `overlay=true` is set